.github/workflows/image-build-pr-check.yml

```diff
@@ -56,10 +56,10 @@ jobs:
         export SKIP_TEST=true
         if [[ "${{matrix.dist}}" == "alpine" ]]; then
           docker pull quay.io/eclipse/che-plugin-registry:nightly
-          ./build.sh --tag pr-check
+          BUILDER=docker ./build.sh --tag pr-check
```

Comment:
Might be interesting to have in Matrix job a build with podman and buildah as well for PR checks

Reply:
`docker` is used to run the containers, so we'd have to run them with `podman`, which seems a bit complicated for a PR check. It's unlikely that something will build on `docker` but fail with `podman`.

Reply:
Let's explore it another time. Okay to merge this PR?

Reply:
sure 
Fine to explore it another time